### PR TITLE
Fibonacci Sequence in JavaScript

### PR DESCRIPTION
**Pull request by MariusGarbea**

### CODE REVIEW

**Comment by jrg94** [comment]
Looks good! Can you make the article issue and link it in the README? There are examples in the README. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: archive/j/javascript/README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Welcome to Sample Programs in JavaScript!
 - [Fizz Buzz in JavaScript][4]
 - [Baklava in JavaScript][8]
   - Solution borrowed from @toturkmen via the [baklava repo][1]
+- [Fibonacci in JavaScript]
```
Comment (jrg94) [inline archive/j/javascript/README.md]: Looks good! Can you make the article issue and link it in the README? There are examples in the README.

**Comment by MariusGarbea** [comment]
Like this?

**Comment by jrg94** [comment]
Looks good! How would I go about testing it?

**Comment by MariusGarbea** [comment]
Unless you have a specific JS interpreter outside of your browser. just paste the code in your console and replace the process.argv[0] with whatever number you want.

**Comment by jrg94** [comment]
True! But then, what's the purpose of the process.argv[0] line?

**Comment by MariusGarbea** [comment]
For consistency reasons with other languages. Also, you can run the code with node just like with any other compiler for a different language.
```
node fibonacci.js 8
```

**Comment by jrg94** [comment]
Sounds great! Thanks for the explanation.

**Pull request merged**